code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -40,6 +41,7 @@
 /datum/reagent/drug/nicotine
 	name = "Nicotine"
 	id = "nicotine"
+	value = 1
```

Comment:
```suggestion
	value = 0
```
Did a quick estimee, even if nerfed, all the nicotine found in the vendor (including premium and contraband) will still net around 2320 (23 packs containing each 6 cigarette/cigars, in turn containing 15 u of nicotine, plus 5 vapes containing 50 u of nicotine each).
In short, this money laundry trick can still be done, just not as effortlessly.